cmd/reports.go

```diff
@@ -36,6 +37,32 @@ var reportsListCmd = &cobra.Command{
 		}
 	},
 }
+var reportsCreateCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Create Report for the given file ID.",
+	Long: `Triggers report generation if report doesn't exists or not updated
```

Comment:
Let not handle 400. As ideally 400 should not occur let the backend handle it if required. For the now the message should be shown to the user

Reply:
Done 👍🏼